packaging/windows/netdata.wxs.in

```diff
@@ -275,6 +271,13 @@
     </Fragment>
 
     <Fragment>
+        <Property Id="OS_HAS_FILE">
+            <!-- Our variables are not defined during startup so we have to use this -->
+            <DirectorySearch Id="NetCoreDirectoryFoundx64" Path="C:\Program Files\Netdata\etc\netdata\">
```

Comment:
Although I can't see the option anymore during installation, what happens if the agent is installed in a different location

Reply:
Good question. This scenario applies to a unique and specific use case where users will specify a variable to run the UI from the command line instead of double-clicking it.

To address the scenario you mentioned, we would need to generate a DLL with a function that can be called within a CustomAction, according to various forums and documentation.

Since this is a corner case, I suggest handling it in a separate PR, as I will need to investigate further into WIX to properly implement it.